### DISCUSSION TITLE
Session storage support

### DISCUSSION THREAD

**Discussion by kemken077** [carbon-design-system/carbon-components-svelte#1731] [Ideas]
Would be nice to have session storage support, right?
not sure if as a separate component or just one single Storage element with both LocalStorage as SessionStorage options.
Please, let me know your thoughts.
I would be glad to help on this.

Thank you!

**Comment by metonym** [comment]
Released in [v0.102.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.102.0).

https://svelte.carbondesignsystem.com/components/SessionStorage
